src/modules/uv-searchfooterpanel-module/FooterPanel.ts

```diff
@@ -132,11 +132,7 @@ export class FooterPanel extends BaseFooterPanel {
     this.$searchPagerContainer.prepend(this.$searchPagerControls);
 
     this.$previousResultButton = $(
-      '<a class="previousResult" title="' +
-        this.content.previousResult +
-        '">' +
-        this.content.previousResult +
-        "</a>"
+      '<a class="previousResult">' + this.content.previousResult + "</a>"
```

Comment:
As these don't have `href` attributes, should we make them buttons?

Reply:
Yes!